src/app/modules/main/activity_center/module.nim

```diff
@@ -107,7 +107,7 @@ proc createMessageItemFromDto(self: Module, message: MessageDto, communityId: st
       imagesAlbum.add(msg.image)
       albumMessageIds.add(msg.id)
 
-  return msg_item_qobj.newMessageItem(msg_item.initItem(
+  return msg_item_qobj.newMessageItem(msg_item.initMessageItem(
```

Comment:
I renamed this function because it was **so** annoying trying to see which function to modify when **every** Item has the same `init` function. Now it's gonna be way clearer.